tsl/test/sql/dist_move_chunk.sql

```diff
@@ -255,6 +255,33 @@ SELECT chunk_schema || '.' ||  chunk_name, data_nodes
 FROM timescaledb_information.chunks
 WHERE hypertable_name = 'dist_test';
 
+DROP TABLE dist_test;
+
+-- Ensure dirty chunks are handled during chunk copy
+CREATE TABLE dist_test(time timestamp NOT NULL, device int, temp float);
+SELECT create_distributed_hypertable('dist_test', 'time', 'device', 3);
+INSERT INTO dist_test SELECT t, (abs(timestamp_hash(t::timestamp)) % 10) + 1, 0.10 FROM generate_series('2018-03-02 1:00'::TIMESTAMPTZ, '2018-03-08 1:00', '1 hour') t;
+
+SELECT * from show_chunks('dist_test');
+SELECT * FROM test.remote_exec(NULL, $$ SELECT * from show_chunks('dist_test'); $$);
+
+-- create dirty chunk
+\c :DATA_NODE_2 :ROLE_CLUSTER_SUPERUSER;
+CREATE TABLE _timescaledb_internal._dist_hyper_4_13_chunk(time timestamp NOT NULL, device int, temp float);
```

Comment:
Had to create it manually, could not found a way to do this right now, before we have a proper detach/drop chunk logic on the AN for HA.

Reply:
@pmwkaa I would recommend to adopt the "stale" chunk nomenclature instead of "dirty" chunk overall for the patch. We can also use "orphan" chunks if you prefer. But in my current work I am using "stale" right now.